{"name": "Получение заказа пользователя без авторизации", "status": "passed", "start": 1709680812404, "stop": 1709680812513, "uuid": "95c078a6-e808-4012-9b9b-6ef467f56b88", "historyId": "4789c7aac39e50078c2e0c4aa7b44d12", "testCaseId": "4789c7aac39e50078c2e0c4aa7b44d12", "fullName": "tests.test_get_order.TestApiGetUserOrders#test_no_auth_user_get_user_order_error", "labels": [{"name": "feature", "value": "Проверяем получение заказа пользователя"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_get_order"}, {"name": "subSuite", "value": "TestApiGetUserOrders"}, {"name": "host", "value": "DESKTOP-AQ71LTK"}, {"name": "thread", "value": "23088-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_get_order"}]}